code/modules/reagents/reagents/reagents_ethanol.dm

```diff
@@ -853,6 +853,35 @@
 	glass_icon_state = "aleglass"
 	glass_desc = "A cold pint of delicious ale."
 
+/datum/reagent/ethanol/drink/cider
+	name = "Cider"
+	id = CIDER
+	description = "Alcoholic, fermented apples."
+	nutriment_factor = 2 * REAGENTS_METABOLISM
+	color = "#ffee88"
+	glass_icon_state = "ciderglass"
+	glass_desc = "The hard kind. Alcoholic."
+
+/datum/reagent/ethanol/drink/scrumpy
+	name = "Scrumpy"
+	id = SCRUMPY
+	description = "Alcoholic, fermented lemons."
+	nutriment_factor = 2 * REAGENTS_METABOLISM
+	color = "#887722"
+	glass_icon_state = "ciderglass"
+	glass_desc = "Ayy... me bottle a scrumpy!"
+
+/datum/reagent/ethanol/drink/stout
+	name = "Stout"
+	id = STOUT
+	description = "Warm fermented alcohol. A good source of iron."
```

Comment:
`>a good source of iron`
`>doesn't give you iron (like medcoffee)`
what did he mean by this

Reply:
Just make it dripfeed you iron, you can copypaste the proc from lifeline (medcoffee).

Reply:
another idea i had was electrolyzing it into iron, but in lieu of it being exclusive to gibness i could do that for now